api/routes.go

```diff
@@ -1,16 +1,26 @@
 package api
 
 import (
+	"context"
 	"net/http"
 	"strings"
 
-	"github.com/SkynetLabs/skynet-accounts/database"
 	"github.com/SkynetLabs/skynet-accounts/jwt"
+	jwt2 "github.com/lestrrat-go/jwx/jwt"
 
 	"github.com/julienschmidt/httprouter"
 	"gitlab.com/NebulousLabs/errors"
 )
 
+var (
+	// APIKeyHeader holds the name of the header we use for API keys. This
+	// header name matches the established standard used by Swagger and others.
+	APIKeyHeader = "X-API-KEY"
```

Comment:
when defining headers I think it's important to domain prefix them, perhaps we should call it `X-SKYNET-API-KEY`?

Reply:
I'd just use `Skynet-Api-Key` to match all our other headers like `Skynet-Disable-Force`, `Skynet-File-Layout`, `Skynet-File-Metadata`, `Skynet-Proof` and `Skynet-Skylink`.

The `X-` convention for custom headers has been deprecated for almost 10 years as part of some RFC.

Reply:
I already mentioned this elsewhere but using `X-API-KEY` makes us compatible with the Swagger implementation of API keys. If we don't want that then we can rename the query param to `apiKey` as well.

Reply:
> I'd just use `Skynet-Api-Key` to match all our other headers like `Skynet-Disable-Force`, `Skynet-File-Layout`, `Skynet-File-Metadata`, `Skynet-Proof` and `Skynet-Skylink`.
> 
> The `X-` convention for custom headers has been deprecated for almost 10 years as part of some RFC.

I didn't see this when writing my reply. Sounds good to me, I'm attached to the header name. Changing.